packages/graphql/src/translate/translate-update.ts

```diff
@@ -67,13 +67,18 @@ export default async function translateUpdate({
     const matchNode = new Cypher.NamedNode(varName);
     const where = resolveTree.args.where as GraphQLWhereArg | undefined;
     const matchPattern = new Cypher.Pattern(matchNode, { labels: node.getLabels(context) });
+    // bypass auth rules if there's no actual update of properties (connect and disconnect have their own auth rules)
+    const areNodePropertiesBeingUpdated = updateInput
+        ? Object.keys(updateInput).filter((x) => !node.relationFields.some((field) => field.fieldName === x)).length > 0
```

Comment:
Is there a reason for not using some in the top level array as well?

```suggestion
        ? Object.keys(updateInput).some((x) => !node.relationFields.some((field) => field.fieldName === x))
```